### ISSUE TITLE
Add title support for slider

### ISSUE THREAD

**Issue by clandmeter** [athombv/homey#1290]
**What is your Homey version (Settings → System)?**

1.1.2

Would be nice if mobile component slider would support a title below. This would come in handy when a mobile card has multiple sliders. See following example:

![image](https://cloud.githubusercontent.com/assets/52970/22218174/09d3e112-e1a6-11e6-9a38-01b9398f8648.png)

I could add the slider to another mobile card, but for my qubino module this is currently not possible, see #1288

**Comment by WeeJeWel** [comment]
Added in Homey v1.1.5! Add a mobile component option `"showTitle": true` to a component with id `slider`

**Comment by clandmeter** [comment]
Not sure im understand this correctly. I added the following to mobile components:

```diff
diff --git a/app.json b/app.json
index f83c22f..225f0d0 100644
--- a/app.json
+++ b/app.json
@@ -3214,13 +3214,19 @@
             "id": "slider",
             "capabilities": [
               "dim.shutter"
-            ]
+            ],
+            "options": {
+              "showTitle": true
+            }
           },
           {
             "id": "slider",
             "capabilities": [
               "dim.venetian"
-            ]
+            ],
+            "options": {
+              "showTitle": true
+            }
           },
           {
             "id": "sensor",
```

Which should add a title under my slider in both browser and mobile device. Am i correct to assume this title should come from the following code:

```json
"capabilitiesOptions": {
  "dim.shutter": {
     "title": {
         "en": "Shutter position"
     }
  },
  "dim.venetian": {
     "title": {
         "en": "Slats position"
     }
  }
},
```

I cannot find any app currently in athoms repos that make use of "showTitle".

**Comment by clandmeter** [comment]
@WeeJeWel I'm lost here, can you help?

**Comment by WeeJeWel** [comment]
`options` must be next to `capabilities`, so in the same object :-)

**Comment by clandmeter** [comment]
@WeeJeWel what is the difference with my code compared to yours? I only see the breaks when settings capabilities?

```json
"mobile": {
	"components": [
		{
			"id": "ternary",
			"capabilities": [
				"windowcoverings_state"
			]
		},
		{
			"id": "slider",
			"capabilities": [
				"dim.shutter"
			],
			"options": {
				"showTitle": true
			}
		},
		{
			"id": "slider",
			"capabilities": [
				"dim.venetian"
			],
			"options": {
				"showTitle": true
			}
		},
		{
			"id": "sensor",
			"capabilities": [
				"measure_power",
				"measure_temperature"
			]
		}
	]
}
```

**Comment by WeeJeWel** [comment]
Ah I guess I didn't follow the diff correctly. This doesn't work for you on Homey v1.1.9?

**Comment by clandmeter** [comment]
![image](https://cloud.githubusercontent.com/assets/52970/23302649/cfec2eac-fa8f-11e6-9586-b266bcb8eca4.png)

I guess it should be on top of the sliders?

**Comment by WeeJeWel** [comment]
Yep. using Homey v1.1.9?

**Comment by clandmeter** [comment]
yes sir!

Homey Firmware Version: 1.1.9

**Comment by WeeJeWel** [comment]
Hm, that should work. Can I download your app somewhere so I can try to reproduce? And can you paste your device's JSON? (Can be read by going into the 'Network' tab in Chrome Inspector)

**Comment by clandmeter** [comment]
Sure, I'm using my branch of the Qubino app patched with the changed without committing them to:

https://github.com/clandmeter/com.qubino/tree/venetian

You can find the diff here:

http://tpaste.us/8Qx6

**Comment by clandmeter** [comment]
@WeeJeWel I cannot find any device json in the inspector, the only json file it fetches is en.json.

The html of the device is:

```html
<div class="component component-slider"><div class="name">Shutter position</div><div class="slider-inner animate"><div class="slider-knob" style="left: 0%;"></div><div class="slider-track"></div><div class="slider-progress" style="width: 0%;"></div></div></div>
```

Looks like the Shutter position name is showing up here.

**Comment by WeeJeWel** [comment]
Fixed in 1.2.0
